tools/test-proxy/Azure.Sdk.Tools.TestProxy/README.md

```diff
@@ -587,3 +597,14 @@ Then, confirm in the right panel that `Development time IIS support` is not chec
 [Add Internet Information](https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/iis/development-time-iis-support?view=aspnetcore-6.0) Services to your Windows installation. Here is the list of features to enable:
 
 ![image](https://user-images.githubusercontent.com/24213737/152258180-0bac3e7f-910c-45fd-aa5f-fc932fce91e6.png)
+
+## Asset Sync (Retrieve External Test Recordings)
+
+The `test-proxy` optionally offers integration with other git repositories for **storing** and **retrieving** recordings. This enables the proxy to work against repositories that do not emplace their test recordings directly alongside their test implementations.
+
+
+
+
```

Comment:
```suggestion

```

Reply:
I hear ya this isn't done by a Lonnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnng shot. More updates coming in this evening.